opendb-core/src/main/java/org/openplacereviews/opendb/service/DBConsensusManager.java

```diff
@@ -859,6 +860,8 @@ public OpBlockChain doInTransaction(TransactionStatus status) {
 			if(conn != null) {
 				try {
 					conn.close();
+					//FIXME
```

Comment:
Why we need this?
throw new SQLException();

Reply:
It's not needed you've just added an unclear comment.

I'm also not sure if connection close is really needed here, probably to make sure that  transaction is closed
but in test it's probably unnecessary